### PR TITLE
Mapper frontend mapper flow

### PR DESCRIPTION
**Pull request by NSUWAL123**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation
- [x] 🧑‍💻 Refactor
- [ ] ✅ Test
- [ ] 🤖 Build or CI
- [ ] ❓ Other (please specify)

## Related Issue
- Related to #1853

## Describe this PR
This PR contains the following works done: 
- UI fix
- Task actions button requiring reload to sync with the task fix
- Code refactor
- Mapper Frontend: Build a mapper flow for first-time project load

## Screenshots
[Screencast from 5-11-24 05:36:49 अपराह्न +0545.webm](https://github.com/user-attachments/assets/485825a9-8bc8-4c02-9574-f09536dc70f9)

### CODE REVIEW

File: src/mapper/src/lib/components/map/main.svelte [hunk 1]
```diff
@@ -1,67 +1,72 @@
...
-    import { polygon } from '@turf/helpers';
-    import { buffer } from '@turf/buffer';
-    import { bbox } from '@turf/bbox';
-    import type { Position, Polygon, FeatureCollection } from 'geojson';
+	import '$styles/page.css';
+	import '$styles/button.css';
+	import '@hotosm/ui/dist/hotosm-ui';
+	import {
+		MapLibre,
+		GeoJSON,
```
Comment (spwoodcock) [inline src/mapper/src/lib/components/map/main.svelte:7]: Are these whitespace changes from an IDE plugin or something? Perhaps a prettier plugin?

Comment (spwoodcock) [reply src/mapper/src/lib/components/map/main.svelte:7]: We should align on a style config to avoid back and forth 👍

**Review by spwoodcock — APPROVED** [review]
Nice!

File: src/mapper/src/lib/components/map/main.svelte (hunk 1 above)
Comment (NSUWAL123) [reply src/mapper/src/lib/components/map/main.svelte:7]: Yeah its from prettier

Comment (spwoodcock) [reply src/mapper/src/lib/components/map/main.svelte:7]: Right! Strange it's not picking up the prettierrc config under src/mapper! Can you copy the values in the plugin somehow?

**Pull request merged**